src/i18n/messages/en.messages.json

```diff
@@ -66,7 +57,7 @@
       "message": "No results found for <span class=\"x-text x-text--accent x-text--bold\">'{query}'</span>. We show you results for"
     },
     "noResults": {
-      "message": "No results found for <span class=\"x-text x-text--accent x-text--bold\">'{query}'</span>. Try with another query."
+      "message": "Sorry but there are no results for <span class=\"x-text x-text--accent x-text--bold\">'{query}'</span>. You may be interested in these products"
```

Comment:
This is not the message in the designs for the `no-results` (no dot after the first phrase and a line break instead). Should be changed here or there's a task for it? (A little styling is required).

![image](https://user-images.githubusercontent.com/7124620/175527327-edd8282d-5e95-4e4a-af5e-737595b1cb7d.png)